### PR TITLE
Adds a widely known wrestler and makes him a toy

### PR DESCRIPTION
**Pull request by Lijah99**
We all know his name is...

### CODE REVIEW

**Comment by Anonus** [comment]
>not using branch names for memes

Terrible! :+1:

**Comment by xxalpha** [comment]
No references.

@Wjohn HARDCORE tag is for HARDCORE branch related things.

**Comment by Jalleo** [comment]
Who? (actually serious)

**Comment by Lijah99** [comment]
https://www.youtube.com/watch?v=58mah_0Y8TU

**Comment by Supermichael777** [comment]
change the name. we have wrestling masks, use those have him say OLE!

**Comment by WJohn** [comment]
We aren't getting anything from the hardcore branch anymore, so now it's just a fun tag to use.

**Comment by duncathan** [comment]
E-excuse me? Is Champ there?

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -637,6 +638,24 @@
...
 	quiet = 1
 
+/obj/item/toy/prize/cenatoy
+	name = "wrestler action figure"
+	desc = "A small figure of a wrestler, but you can't seem to recall his name."
+	icon_state = "cenatoy"
+
+/obj/item/toy/prize/cenatoy/attack_self(mob/user)
+	if(!cooldown)
+		var/message = pick("You can't see me!", "Body slammin' time!", "My name is JOHN CENA!")
```
Comment (Iamgoofball) [inline code/game/objects/items/toys.dm]: Macho man or bust faggot

**Comment by Iamgoofball** [comment]
This is dumb because cena aint got nothin on macho man Randy Savage

**Comment by Lijah99** [comment]
Hey brother, I'll snap you like a slim-jim :^)

**Comment by optimumtact** [comment]
Hey Champ, good work

**Comment by JJRcop** [comment]
I was actually excited for another hardcore PR but then I found out the tag was misused... :(

**Comment by Lijah99** [comment]
I'm disappointed, no one has yet guessed his name! :-1:

**Comment by Iamgoofball** [comment]
@Lijah99 https://www.youtube.com/watch?v=8C4lK41SX-Q

**Comment by optimumtact** [comment]
His name? Potatoe Salad

**Comment by whitefox08** [comment]
0/10 no macho taco randy sandwich

**Comment by Miauw62** [comment]
this meme is going to stop being funny in a few months and we will have a shitty dank meme toy for an old meme nobody likes anymore

**Comment by Miauw62** [comment]
actually ill reopen this so people cant call me hypocritical when i add other dank memes to the game

**Comment by Lijah99** [comment]
):

**Comment by Niknakflak** [comment]
:-1:

**Comment by optimumtact** [comment]
:+1:

**Comment by Shadowlight213** [comment]
:-1: I can't believe I'm agreeing with goof here. Macho man or bust.

**Comment by SconesC** [comment]
macho is an OLD and DEAD meme run into the ground by players

**Comment by Iamgoofball** [comment]
That's john cena, macho man is the ss13 wrestler meme.

**Comment by Lijah99** [comment]
What goofball said, but we all know John Cena is superior.

**Comment by Iamgoofball** [comment]
nope, macho man could out-wrestle john cena

**Comment by Lijah99** [comment]
But you can't wrestle someone if you "can't see him"
kek

**Comment by Iamgoofball** [comment]
the macho madness has ways

**Comment by Dorsisdwarf** [comment]
Make him MACHO MAN, that means I can fire the HOP as captain for impersonating a famous wrestler

**Comment by phil235** [comment]
:-1:  The only use for Toy figures is to reference something in-game (griffin action figure, mech toys, AI toy). This proposed toy doesn't reference anything ingame, I don't see any reason why we should have it. If we accept it, it would be hypocritical to refuse other pop culture references toy.

**Comment by Lijah99** [comment]
Well jeez

**Comment by Lijah99** [comment]
If that Metal Gear Solid me-me was added, is this more horrific?

**Comment by freerealestate** [comment]
this one's been played out for years

the original prank call that kicked it off is at least as old as 2012 http://media.z104.com/a/67368183/phone-scam-john-cena.htm

mgsv is less than a month old

**Comment by Lijah99** [comment]
That still doesn't justify using the argument, "no memes" on one item as you integrate another, which is indeed a meme

**Comment by Iamgoofball** [comment]
the argument isn't no memes

the argument is "some memes are shittier than others"

**Comment by Aranclanos** [comment]
no thanks.

**Comment by Iamgoofball** [comment]
:+1:

**Comment by Miauw62** [comment]
box memes arent based on mgsv, the original metal gear game had a cardboard box.

john cena has only really been a meme for a few months.

i dont really want either in the game, tbh, at least not in the obnoxious way the box meme has been implemented.

**Comment by Aranclanos** [comment]
you're all a bunch of hypocrites though

**Comment by Iamgoofball** [comment]
You aren't listening

**Comment by Lijah99** [comment]
I thoroughly agree with Aran. If you can't see that what you've done is hypocritical, you're all blind.

**Comment by freerealestate** [comment]
yeah but mgsv has renewed interest in mgs roleplaying, @Miauw62

>john cena has only really been a meme for a few months.

i guess it's subjective, but i've been aware of it for a year or two now.

>the obnoxious way the box meme has been implemented.

i like the box. its sprite is pretty damn ugly but it serves a practical in-game use (moving while untrackable by the AI/passing by people without them IDing you) and fits into the environment (as it's just a cardboard box)

**Pull request closed without merging**